### DISCUSSION TITLE
Custom HTTP headers

### DISCUSSION THREAD

**Discussion by BenasB** [testcontainers/testcontainers-dotnet#568] [Q&A] [answered]
Do you plan to allow custom HTTP headers when making requests? I can see this PR in Docker.DotNet: https://github.com/dotnet/Docker.DotNet/pull/580 but I'm not sure when that code change will get released.

I'd use this feature for adding an HTTP Cookie to my requests to a docker host behind a load balancer (for load balancer session persistence).

I could try to make a PR for this. Thanks!

**Comment by HofmeisterAn** [comment, marked as answer]
> Do you plan to allow custom HTTP headers when making requests?

Yes, you will be able to set custom HTTP headers in the future.
